qualtran/bloqs/multi_control_multi_target_pauli.py

```diff
@@ -11,27 +11,29 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-
+from functools import cached_property
 from typing import Tuple
 
 import cirq
 import numpy as np
+import sympy
 from attrs import field, frozen
-from cirq._compat import cached_property
 from numpy.typing import NDArray
 
-from qualtran import GateWithRegisters, Signature
+from qualtran import bloq_example, BloqDocSpec, GateWithRegisters, Signature
 from qualtran.bloqs.and_bloq import And, MultiAnd
 from qualtran.cirq_interop.t_complexity_protocol import t_complexity, TComplexity
 
 
 @frozen
 class MultiTargetCNOT(GateWithRegisters):
-    """Implements single control, multi-target CNOT_{n} gate in 2*log(n) + 1 CNOT depth.
+    r"""Implements single control, multi-target $C[X^{\otimes n}]$ gate.
+
+    Implements $|0><0| I + |1><1| X^{\otimes n}$ using a circuit of depth $2\log(n) + 1$
```

Comment:
Go all the way and use langle / rangle?

Reply:
trying to strike a balance between readability in the file vs in the notebook. I've been secretly harboring a desire to add a pre-processor to the notebook rendering to turn |...> into `\rangle`s